cmake/compilers/Clang.cmake

```diff
@@ -65,6 +65,10 @@ endif()
 # Clang flags to prevent compiler from optimizing out security checks
 set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} -Wformat -Wformat-security -Werror=format-security -fPIC $<$<NOT:$<BOOL:${EMSCRIPTEN}>>:-fstack-protector-strong>)
 
+if (NOT APPLE AND NOT ANDROID_PLATFORM)
+    set(TBB_LIB_COMPILE_FLAGS ${TBB_LIB_COMPILE_FLAGS} -fstack-clash-protection $<$<NOT:$<BOOL:${EMSCRIPTEN}>>:-fcf-protection=full>)
+endif()
```

Comment:
> * -fcf-protection=full flags - not supported in Emscripten and on aarch64.

Won't this be adding `-fcf-protection=full` on aarch64 on Windows? Or is that a GCC-only limitation?

Reply:
> > * -fcf-protection=full flags - not supported in Emscripten and on aarch64.
> 
> Won't this be adding `-fcf-protection=full` on aarch64 on Windows? Or is that a GCC-only limitation?

Unfortunately, I can't verify it with Clang on Windows arm64.